bin/cron_hpc_at_cms.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# This script is intended to be used as cron job. 
+# For this reason it will use sensible defaults 
+# for the required parameters.
+if [ "$1" = "-h" ] || [ "$1" = "--help" ]
+then
+cat <<EOF
+ Usage: cron_hpc_at_cms.sh <OUTPUT_DIR> <START_DATE> <END_DATE>
+   - if OUTPUT_DIR is not specified, $HOME/output_hpc_at_cms will be used.
+   - if not END_DATE is specified, it will use the first day of the current month. 
+   - if there is not START_DATE and END_DATE it will cover a 1 year period,
+     ending in the first day of the current month (without including it)
+EOF
+exit 0
+fi
+OUTPUT_DIR="${1:-$HOME/output_hpc_at_cms}"
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
+
+END_DATE="${3:-$(date +%Y-%m-01)}"
+START_DATE="${2:-$(date -d "$END_DATE -1 year" +%Y-%m-01)}"
```

Comment:
both days should be validated first before passing to the script.

Reply:
I saw later that python script calls the validation. I suggest that you add a comment here in  a script that actual validation will be performed by python script.